### PR TITLE
[Batch] Update Management API used

### PR DESCRIPTION
**Pull request by bgklein**

### CODE REVIEW

File: sdk/batch/azure-mgmt-batch/azure/mgmt/batch/models/_paged_models.py [hunk 1]
```diff
@@ -25,8 +25,6 @@ class BatchAccountPaged(Paged):
     def __init__(self, *args, **kwargs):
 
         super(BatchAccountPaged, self).__init__(*args, **kwargs)
-
```
Comment (xingwu1) [inline sdk/batch/azure-mgmt-batch/azure/mgmt/batch/models/_paged_models.py:28]: manually add back these empty lines.

File: sdk/batch/azure-mgmt-batch/azure/mgmt/batch/models/_paged_models.py [hunk 2]
```diff
@@ -85,8 +77,32 @@ class CertificatePaged(Paged):
     def __init__(self, *args, **kwargs):
 
         super(CertificatePaged, self).__init__(*args, **kwargs)
+class PrivateLinkResourcePaged(Paged):
```
Comment (xingwu1) [inline sdk/batch/azure-mgmt-batch/azure/mgmt/batch/models/_paged_models.py:80]: add 2 empty lines.

File: sdk/batch/azure-mgmt-batch/azure/mgmt/batch/operations/_batch_account_operations.py [hunk 3]
```diff
@@ -162,19 +162,17 @@ def get_long_running_output(response):
     create.metadata = {'url': '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Batch/batchAccounts/{accountName}'}
 
     def update(
-            self, resource_group_name, account_name, tags=None, auto_storage=None, custom_headers=None, raw=False, **operation_config):
```
Comment (xingwu1) [inline sdk/batch/azure-mgmt-batch/azure/mgmt/batch/operations/_batch_account_operations.py:165]: where is tag and auto_storage?
I didn't see any change in BatchAccountUpdateParameters for tag and auto_storage

Comment (bgklein) [reply sdk/batch/azure-mgmt-batch/azure/mgmt/batch/operations/_batch_account_operations.py:165]: The auto gen must have not honored           
"x-ms-client-flatten": true,
          "$ref": "#/definitions/BatchAccountProperties",
          "description": "The properties associated with the account."

**Pull request merged**